distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+chmod -R og-w ${branding}_HOME/etc ${branding}_HOME/data
+----
++
+. Restrict access to sensitive files by ensuring that the only users with “group” permissions (e.g., ${branding-lowercase}-group) have access.
+. Execute the following command on the above files (examples assume `${branding}_HOME` is `/opt/${branding-lowercase}`):
++
+[source,bash]
+----
+chmod -R o /opt/${branding-lowercase}
+----
++
+. As the application owner (e.g., ${branding-lowercase} user), restrict access to sensitive files.
++
+[source,bash,linenums]
+----
+chmod 640 /opt/${branding-lowercase}/etc
+chmod 640 /opt/${branding-lowercase}/deploy
+----
+
+[IMPORTANT]
+====
+The system administrator must restrict certain directories to ensure that the application (user) cannot access restricted directories on the system.
+For example the `NON_ROOT_USER` should only have read access to `/opt/${branding-lowercase}`.
+====
+
+==== Java Requirements
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+*
+.Setting `JAVA_HOME` on *NIX
+----
+JAVA_HOME=/usr/java/jdk1.8.0
+export JAVA_HOME
+----
+
+.Setting `JAVA_HOME` on Windows
+----
+set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
+----
+
+.*NIX
+[WARNING]
+====
+Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
+`unlink /usr/bin/java`
+====
+
+* Verify that the `JAVA_HOME` was set correctly.
+
+.*NIX
+----
+echo $JAVA_HOME
+----
+
+.Windows
+----
+echo %JAVA_HOME%
+----
+
+* ${branding} installation zip file.
+* A web browser.
+* For Linux systems, increase the file descriptor limit by editing `/etc/sysctl.conf` to include or change the following:
+
+.File Descriptor Limit
+----
+fs.file-max = 6815744
+----
+
+.Restart
+[WARNING]
+====
+For the change to take effect, a restart is required.
+====
+
+.Restart Command
+----
+init 6
+----
+
+==== Deployment Guidelines
+
+${branding} relies on the Directory Permissions of the host platform to protect the integrity of the ${branding} during operation.
+System administrators MUST perform the following steps prior to deploying bundles added to the ${branding}.
+
+. Check the available storage space on the system to ensure the deployment will not exceed the available space.
+. Set maximum storage space on the `${branding}_HOME/deploy` and `${branding}_HOME/system` directories to restrict the amount of space used by deployments.
+. Do not assume the deployment is from a trusted source; verify its origination.
+. Use the source code to verify a deployment is required for ${branding} to prevent unnecessary/vulnerable deployments.
```

Comment:
Not sure either. Dropped it. 

I think it might've meant "Make sure you only install the components you need" but it's hard to tell.